src/main/java/com/ericsson/ei/controller/RuleTestControllerImpl.java

```diff
@@ -85,8 +86,13 @@
             final HttpServletRequest httpRequest) {
         if (testEnabled) {
             try {
-                String aggregatedObject = ruleTestService.prepareAggregatedObject(
-                        new JSONArray(body.getListRulesJson()), new JSONArray(body.getListEventsJson()));
+                String aggregatedObject = StringUtil.EMPTY_STRING;
+                try {
+                        aggregatedObject = ruleTestService.prepareAggregatedObject(
+                                new JSONArray(body.getListRulesJson()), new JSONArray(body.getListEventsJson()));
+                } catch (Exception e) {
+                    e.printStackTrace();
```

Comment:
donot print stacktrace, use loggers to log them

Reply:
Corrected.